src/components/WorkspaceSelectionDialog.js

```diff
@@ -28,41 +43,70 @@ export class WorkspaceSelectionDialog extends Component {
    */
   render() {
     const {
-      classes, container, handleClose, open, children, t,
+      classes, container, handleClose, open, children, t, workspaceType,
     } = this.props;
     return (
       <Dialog
         aria-labelledby="workspace-selection-dialog-title"
         container={container}
-        id="workspace-settings"
+        id="workspace-selection-dialog"
         onClose={handleClose}
+        onEscapeKeyDown={handleClose}
         open={open}
       >
-        <DialogTitle id="workspace-selection-dialog-title">{t('workspaceSelectionTitle')}</DialogTitle>
+        <DialogTitle id="workspace-selection-dialog-title" disableTypography>
+          <Typography variant="h2">{t('workspaceSelectionTitle')}</Typography>
+        </DialogTitle>
         <DialogContent>
           {children}
-          <List>
+          <ListKeyboardNavigation
+            className={classes.list}
+            onChange={this.handleworkspaceTypeChange}
+            selected={workspaceType}
+          >
             <ListItem
               className={classes.listItem}
-              onClick={() => this.handleworkspaceTypeChange('elastic')}
+              value="elastic"
             >
-              <img src="/src/images/elastic.jpg" alt={t('elastic')} />
-              <ListItemText
-                primary={t('elastic')}
-                secondary={t('elasticDescription')}
-              />
+              <Card className={classes.card}>
+                <CardMedia
+                  className={classes.media}
+                  image="/src/images/icon_workspace_elastic.svg"
```

Comment:
I'm concerned that referencing the image this way will hardcode the image path (and its not included in the build). Is there a way to reference the SVG directly?

Reply:
@mejackreed Do we have a pattern established for this anywhere else in the repo?

Reply:
Yeah, I think the predominate pattern is to wrap the SVG in a component like we do with the icons:

https://github.com/ProjectMirador/mirador/blob/master/src/components/icons/BookViewIcon.js

Reply:
@ggeisler Could you provide the two SVGs in a scaled variant (90x120)? Because resizing the files with inkscape seems not to work properly.

Reply:
@glefi I've updated the gists I previously gave you, so try those and see how they work:

- Mosaic: https://gist.github.com/ggeisler/21c514a51ae9eb0cf62288a1f05e6ca0
- Elastic: https://gist.github.com/ggeisler/3bacfde8f287748603a6bc33e0ef8197